docs/updates/2019-03.md

```diff
@@ -34,13 +38,38 @@ To increase usability when loading large applications (or with slow connections)
 
 We're in the last phases of finishing a 1.0.0 version of corectl. Focus of this release is to remove the experimental
 flag and make the command structure stable. Along with necessary consistency changes in the API we are adding a
-couple of requested features like bash completion, support for nested JSON structures and better error messages.
+couple of requested features in the latest 0.5.0 release.
+
+### Auto completion
+
+To simplify entering commands, arguments and flags, we have added support for bash completion in bash and zsh. Apart
+from speeding up when entering commands, it lists available objects for `get` commands (dimension, measure, connection
+and app). This will save one call to corectl to retrieve the IDs of the object.
```

Comment:
`get app` is not a command. Bash completions was however also added for `get object ...`.